### PR TITLE
3: QA Review

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Rename `2: QA Review` to `3: QA Review` and do the following migrations.

I can be applied using `labels  --migrate --core-labels --repo $REPO_NAME` to a specific repository or using `labels  --migrate --core-labels` to all pre defined repositories. 

#### Ticket Link
https://community.mattermost.com/core/pl/eegcso8xr3bqzr7giftc3dgdka

### CODE REVIEW

File: labels/mapping.go [hunk 1]
```diff
@@ -107,4 +106,8 @@ var migrateMap = map[string]string{
 	"Enhancement":     "Type/Enhancement",
 	"Question":        "Type/Question",
 	"Tech/JavaScript": "",
+	// Migrate QA labels: https://community.mattermost.com/core/pl/eegcso8xr3bqzr7giftc3dgdka
+	"2: QA Review":        "3: QA Review",
+	"3: Reviews Complete": "4: Reviews Complete",
+	"QA Review Done":      "",
```
Comment (lieut-data) [inline labels/mapping.go]: I'd propose we leave this one untouched for now, and let QA iterate on it separately.

**Review by lieut-data — CHANGES_REQUESTED** [review]
Nice!! One comment below.

**Review by lieut-data — APPROVED** [review]

**Review by levb — APPROVED** [review]

**Comment by levb** [comment]
@hanzei Why DNM?

**Comment by hanzei** [comment]
@levb [Dark Net Markets?](https://www.urbandictionary.com/define.php?term=DNM)

File: labels/mapping.go (hunk 1 above)
Comment (saturninoabril) [reply labels/mapping.go]: Thanks.

**Comment by levb** [comment]
Do Not Merge

**Comment by hanzei** [comment]
I'm waiting on https://github.com/mattermost/mattermost-developer-documentation/pull/584 to get merged.

**Comment by lieut-data** [comment]
@hanzei, this is ready to go now -- deferring to your merging in case it also needs to be "run" somehow.

**Comment by hanzei** [comment]
Merged and applied.

**Pull request merged**